tests/manage/pv_services/test_pvc_disruptive.py

```diff
@@ -228,12 +228,22 @@ def pods_creation(self, pvc_objs, pod_factory, interface):
 
         # Create one pod using each RWO PVC and two pods using each RWX PVC
         for pvc_obj in pvc_objs:
+            if pvc_obj.volume_mode:
+                pod_dict = constants.CSI_RBD_RAW_BLOCK_POD_YAML
+                raw_block_pv = True
```

Comment:
is there any possibility that user can pass access_modes that includes volume mode other than block ? Here we are not checking what type of volume mode is present in pvc_obj.volume_mod and directly assigning raw_block_pv = True

Reply:
We are specifying volume mode only if it is Block. So there is no other combination in this testcase. The script is modified because of the reason mentioned in [previous comment](https://github.com/red-hat-storage/ocs-ci/pull/901#discussion_r335102144)